src/module/Ui/src/Ui/View/Helper/RandomBanner.php

```diff
@@ -26,6 +26,7 @@
 
 class RandomBanner extends AbstractHelper
 {
+    // FIXME: Is this still in use? If not, please delete (including files in src/public. If it is, move images to athene2-assets/packages/assets
```

Comment:
currently not. I think we just kept it there in case we want to activate it again some day. But we could still recover through git if necessary, so I'm fine with deleting it^^